### DISCUSSION TITLE
Pick an account dialog on multi user system

### DISCUSSION THREAD

**Discussion by ASMailer** [Micke-K/IntuneManagement#145] [General]
Hey Mike,

i have a question regarding this dialog below. On a multi user system it shows the account names in the list that have signed in before.
Is there a possibility to clean this, or prevent this? Thanks in advance. Greetings Andi

![image](https://github.com/Micke-K/IntuneManagement/assets/79043883/c17b69d1-9f6f-4728-b660-50ec51e67b30)

**Comment by Micke-K** [comment]
Hello!

This is handled by MSAL / browser cache. I looked into this with the original version of MSAL I had but there was no solution for it.

I might have a look if they have added anything for it in a later version.

Cheers!
